src/classes/TDTM_RunnableMutableExample_TEST.cls

```diff
@@ -0,0 +1,60 @@
+/**
+ * @author Roger Mitchell
+ * @date 4/12/2019
+ * @group TDTM
+ * @group-content ../../ApexDocContent/TDTM.htm
+ * @description Sample implementation of TDTM_RunnableMutable to allow for testing
+ * expected behavior and assisting with test coverage of TDTM_RunnableMutable. For
+ * each time this class runs, the expectation is that the Account's NumberOfEmployees
+ * field will increment by 1.
+ */
+@isTest
+public class TDTM_RunnableMutableExample_TEST extends TDTM_RunnableMutable {
+    
+    public override void run(List<SObject> newList, List<SObject> oldList, 
```

Comment:
Do you think this could be accomplished with a `System.mock` instead of as a separate class?

Reply:
@force2b I think this implementation is more appropriate considering `Test.setMock` is typically reserved for callouts.

Reply:
Sorry, mean to says `System.stub`. I was thinking more about using `System.StubProvider' to mock the class and functionality. In the end though it might still come out the same and not provide any extra capability since both are extending from a global abstract class. A mocked version of this would look pretty much the same as this class that extends the global abstract version. Maybe the only advantage might be that you could have this as an inner class in the `TDTM_RunnableMutable_TEST` class.

Reply:
@force2b keeping it as a separate class will likely cut down effort for your documentation team, as it exists as a standalone example (provided you update scope to `global` and remove `@isTest`.